includes/class-comment.php

```diff
@@ -389,13 +389,17 @@ public static function get_source_url( $wp_comment_id, $fallback = true ) {
 	 * @return string $url
 	 */
 	public static function remote_comment_link( $comment_link, $comment ) {
+		$remote_comment_link = null;
+
 		if ( ! $comment || is_admin() ) {
 			return $comment_link;
 		}
 
```

Comment:
```suggestion
		if ( ! $comment || is_admin() ) {
			return $comment_link;
		}

		$remote_comment_link = null;
```
Let's wait with defining it until we need it?